docs/mobile/overview.mdx

```diff
@@ -10,7 +10,7 @@ import DesignerCallout from "./snippets/designer-callout.mdx";
 
 Geocortex Mobile is a framework for configuring and building mobile offline-capable custom apps on Esri's platform. Geocortex Mobile is part of the [Geocortex suite of products](https://www.geocortex.com/products/).
 
-You can use the Geocortex Mobile framework to build mobile, offline-capable apps that can run on Android, iOS, or Windows platforms. Geocortex Mobile apps make it possible to pan and zoom the map, search for features, review attributes, add and edit features, add attachments, and track your movements in real time. Integration with [Geocortex Workflow](../workflow/overview.mdx) ensures that you can build apps for many business processes. Geocortex Mobile apps can be used in the field to collect data, including photos and documents that can be attached to the map, perform inspections, or add and edit ArcGIS features.
+You can use the Geocortex Mobile framework to build mobile, offline-capable apps that can run on Android, iOS, or Windows platforms. Geocortex Mobile apps make it possible to pan and zoom the map, search for features, review attributes, add and edit features, add attachments, and track your movements in real time. Integration with [Geocortex Workflow](../workflow/overview.mdx) ensures that you can build apps for many business processes. Geocortex Mobile apps can be used in the field to collect data, including photos and documents, perform inspections, or add and edit ArcGIS features.
```

Comment:
This text was originally stolen from https://docs.geocortex.com/mobileviewer/latest/admin-help/Default.htm#gmv/designer/introduction-to-GXM.htm#Introduction_to_Geocortex_Mobile_Designer_Help%3FTocPath%3DIntroduction%2520to%2520Geocortex%2520Mobile%2520Designer%2520Help%7C_____0 so I'm tempted not to change it

Reply:
Reverted to the original.